e2e/utils.go

```diff
@@ -1091,7 +1091,7 @@ func validateController(f *framework.Framework, pvcPath, appPath, scPath string)
 	pv.ResourceVersion = ""
 	err = createPVCAndPV(f.ClientSet, pvc, pv)
 	if err != nil {
-		e2elog.Failf("failed to create PVC or PV with error %v", err)
+		e2elog.Failf("failed to create PVC or PV: %v", err)
```

Comment:
`%w` here?

Reply:
No, `%w` is for returning a wrapped error, not for logging an error.

Reply:
Yeah, sorry. Didn't notice that this was with logging.